.circleci/config.yml

```diff
@@ -38,7 +38,7 @@ jobs:
         command: |
           rustc --version
           cargo --version
-          cargo test --all --all-features
+          cargo test --all --all-features -- --test-threads 1
```

Comment:
A simpler (or faster) way to make sure tests-files do not interfere with each other would be to use different chain-ids per test